pkg/types/conf.go

```diff
@@ -218,6 +219,32 @@ func CreateCNIRuntimeConf(args *skel.CmdArgs, k8sArgs *K8sArgs, ifName string, r
 		},
 	}
 
+	// get CNI_ARGS and set it if it is not exists in rt.Args
```

Comment:
```suggestion
        // get CNI_ARGS and set it if it does not exist in rt.Args
```

Or just drop the comment.

Reply:
fixed.